distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -568,13 +565,7 @@ type Vector a
 
             [[1, 2, 3], [4, 10], [], [0], [0]] . flatten == [1, 2, 3, 4, 10, 0, 0]
     flatten : Vector Any
-    flatten self =
-        length = self.fold 0 acc-> elem-> acc + elem.length
-        arr = Array.new length
-        self.fold 0 i-> vec->
-            Array.copy vec.to_array 0 arr i vec.length
-            i + vec.length
-        Vector.from_polyglot_array arr
+    flatten self = @Builtin_Method "Vector.flatten"
```

Comment:
I am surprised flatten isn't written in Enso and requires a builtin. Less builtins is always better...
```haskell
flat x =
    size = x.map .length . fold 0 (+)
    b = Vector.new_builder size
    x.map arr->
      arr.map e->
        b.append e
    b.to_vector
```
this code seems to `flatten` the elements too.